### PR TITLE
onboard codeready-toolchain registration-service

### PR DESCRIPTION
**Pull request by nurali-techie**
Fixes: https://jira.coreos.com/browse/CRT-317

### CODE REVIEW

File: ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  container:
+    from: src
+- as: test
+  commands: make test #-with-coverage upload-codecov-report
+  container:
+    from: src
+# - as: e2e
+#   commands: make test-e2e
+#   openshift_installer_src:
+#     cluster_profile: aws
```
Comment (nurali-techie) [inline ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml]: @dipak-pawar can we enable this now ??  does this required to have `make test-e2e` target available in registration_service first ??  As of now `make test-e2e` is not available with reg_service.

Comment (dipak-pawar) [reply ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml]: we can enable this, and later we can add required things for e2e on registration service side.

Comment (nurali-techie) [reply ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml]: @dipak-pawar thanks, enabling all target (whatever there for toolchain-operator pr) and later will add required target in reg_service, commit https://github.com/openshift/release/pull/5539/commits/8b96c66a5b4553f34679a9e3f50a30df1c9151f9

**Comment by nurali-techie** [comment]
@dipak-pawar @alexeykazakov @MatousJobanek @xcoulon plz provide your review.  Few checks are failing due to missing make target in registration service.

File: ci-operator/config/codeready-toolchain/registration-service/OWNERS [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
```
Comment (dipak-pawar) [inline ci-operator/config/codeready-toolchain/registration-service/OWNERS]: @nurali-techie Can you please remove `michaelkleinhenz`, as he is not part of openshift organisation. I have already asked him to do the necessary steps for it.

File: ci-operator/config/codeready-toolchain/registration-service/OWNERS [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- nurali-techie
+- sbryzak
+- tinakurian
+- xcoulon
+options: {}
+reviewers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
```
Comment (dipak-pawar) [inline ci-operator/config/codeready-toolchain/registration-service/OWNERS]: Can you please remove michaelkleinhenz, as he is not part of openshift organisation. I have already asked him to do the necessary steps for it.

File: ci-operator/jobs/codeready-toolchain/registration-service/OWNERS [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
```
Comment (dipak-pawar) [inline ci-operator/jobs/codeready-toolchain/registration-service/OWNERS]: Can you please remove michaelkleinhenz, as he is not part of openshift organisation.

File: ci-operator/jobs/codeready-toolchain/registration-service/OWNERS [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- nurali-techie
+- sbryzak
+- tinakurian
+- xcoulon
+options: {}
+reviewers:
+- alexeykazakov
+- dipak-pawar
+- matousjobanek
+- michaelkleinhenz
```
Comment (dipak-pawar) [inline ci-operator/jobs/codeready-toolchain/registration-service/OWNERS]: Can you please remove michaelkleinhenz, as he is not part of openshift organisation. I have already asked him to do the necessary steps for it.

File: ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+- as: build
+  commands: make build
+  container:
+    from: src
+- as: lint
+  commands: make lint-yaml
+  container:
+    from: src
+- as: test
+  commands: make test-with-coverage upload-codecov-report
```
Comment (dipak-pawar) [inline ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml]: Make sure to add `test-with-coverage` and `upload-codecov-report` make targets in registration-service repository

Comment (nurali-techie) [reply ci-operator/config/codeready-toolchain/registration-service/codeready-toolchain-registration-service-master.yaml]: Adding with PR https://github.com/codeready-toolchain/registration-service/pull/51

**Comment by stevekuznetsov** [comment]
Let's make sure the bots have appropriate permissions and branch protection is off

File: ci-operator/config/codeready-toolchain/registration-service/OWNERS (hunk 2 above)
Comment (nurali-techie) [reply ci-operator/config/codeready-toolchain/registration-service/OWNERS]: `michaelkleinhenz` is now part of openshift org.

**Pull request merged**